src/tests/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(unit_tests_antares_xpansion NONE )
+
+find_package(Python3 EXACT 3.8 COMPONENTS Interpreter)
+if(Python3_Interpreter_FOUND)
```

Comment:
This condition is also checked in function `find_python_module`, consider removing one check